src/DistributedMemcacheMutex.php

```diff
@@ -28,16 +30,28 @@ public function __construct(Memcache $memcache)
      * @param string $name
      * @param int $expirationTimeInSeconds
      *
+     * @throws InvalidArgumentException
      * @throws AlreadyLockedException
      */
     public function lock($name, $expirationTimeInSeconds = self::FIVE_MINUTES_SECONDS)
     {
+        $this->assertServersAddedToMemcache();
```

Comment:
Please move to constructor. Also, have you considered using `$memcache->getVersion`? It might perform better (might not, too).